examples/gno.land/r/demo/voting/simple_voting.gno

```diff
@@ -0,0 +1,191 @@
+package voting
+
+import (
+	"errors"
+	"std"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/bitmap"
+	"gno.land/p/demo/nonce"
+	"gno.land/p/demo/seqid"
+)
+
+// Proposal represents a voting proposal with associated metadata and voting state
+type Proposal struct {
+	owner           std.Address    // Owner is the address of the user who created the proposal
+	title           string         // Title is the name or description of the proposal
+	votes           uint64         // Votes is the total number of votes the proposal has received
+	voters          *bitmap.Bitmap // Voters is a bitmap tracking whether users have voted
+	userVoteCount   *nonce.Manager // UserVoteCount maps user addresses to the number of votes they have cast
```

Comment:
You can use an `avl.Tree // std.Address -> *int` without needing to update the tree each time you increment the pointed integer.